flexmeasures/cli/testing.py

```diff
@@ -141,8 +133,8 @@ def test_generic_model(
             model_identifier,
             fallback_model_identifier,
         ) = linear_model_configurator(
-            sensor=old_sensor.corresponding_sensor,
-            time_series_class=determine_old_time_series_class_by_old_sensor(old_sensor),
+            sensor=sensors[0],
+            time_series_class=TimedBelief,  # todo: requires streamlining of the collect function on old sensor data classes with the search function on the TimedBelief class
```

Comment:
Are we now creating `TimedBeliefs` forecasts for assets, which still have their measurements in one of the `TimedValue` tables? That would not seem like a sensible state.

Reply:
You're right. I should move this a little later, together with moving over the old sensor data models. I had permitted myself this exception, because this concerns a commented out CLI function for manual developer tests. But I fixed it now.